### PR TITLE
RFC: Creating Issue Templates

### PR DESCRIPTION
**Pull request by endingwithali**
I added issue templates to the issue - this encourages community members to better organize their thoughts as well as creates instructions for those looking for help to check out our discord.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Get support in our Discord
+    url: https://coder.com/chat
+    about: Need help debugging issues with Coder? Check out our Discord to chat live with the team.
```
Comment (bpmct) [inline .github/ISSUE_TEMPLATE/config.yml:6]: 🙌🏼

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+<!--- Provide a general summary of the requested feature in the Title above -->
+
+## Expected Behavior
+
+<!--- Tell us what should happen for this new feature, including potential pictures-->
+
+## Tasks
+- [ ] <!--- Create a potential task list here -->
```
Comment (bpmct) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]: I'm not sure if the task list is appropriate for the community to fill in here

File: cli/cliui/table.go [hunk 3]
```diff
@@ -67,7 +67,7 @@ func DisplayTable(out any, sort string, filterColumns []string) (string, error)
 	}
 
 	// Get the list of table column headers.
-	headersRaw, err := typeToTableHeaders(v.Type().Elem())
```
Comment (bpmct) [inline cli/cliui/table.go:70]: looks like you might want to rebase from main to remove these commits

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+name: "Feature Request"
+about: Use this form to request new features to be added to the Coder product
+title: 'Feature: '
```
Comment (bpmct) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]:
```suggestion
title: 'Feature: '
```

I think we're trying to remove away from prefixes for bugs & features

File: .github/ISSUE_TEMPLATE/external_bug_report.md [hunk 5]
```diff
@@ -1,3 +1,11 @@
+---
+name: "🐜 Bug Report"
+about: Report a reproducible bug or regression.
+title: 'Bug: '
```
Comment (bpmct) [inline .github/ISSUE_TEMPLATE/external_bug_report.md:4]: I think we're trying to remove away from prefixes for bugs & features

**Review by bpmct — COMMENTED** [review]
Left some comments - can approve once the go code is out

File: cli/cliui/table.go (hunk 3 above)
Comment (endingwithali) [reply cli/cliui/table.go:70]: yes i need to remove these my bad.

**Comment by ammario** [comment]
We had issue templates for a while then decided to remove them since we don't want to inhibit feedback. I think we should hold off on templates until we have the problem of poorly made issues.

**Pull request closed without merging**